packages/select/src/components/select/suggest.tsx

```diff
@@ -284,6 +285,23 @@ export class Suggest<T> extends React.PureComponent<ISuggestProps<T>, ISuggestSt
         Utils.safeInvokeMember(this.props.popoverProps, "onOpened", node);
     };
 
+    private handlePopoverClosed = (node: HTMLElement) => {
+        // The activeItem should always be the selectedItem when the Popover is first opened
+        // if the activeItem prop is not set. Set the activeItem on close so that there isn't
+        // a flash of the activeItem on screen.
+        const shouldResetActiveItemToSelectedItem =
+            this.props.activeItem === undefined && this.state.selectedItem && !this.props.resetOnSelect;
+
+        if (this.queryList && shouldResetActiveItemToSelectedItem) {
+            // If the selectedItem prop is set then use it.
+            // If not fall back to component state.
+            const selectedItem = this.props.selectedItem || this.state.selectedItem;
```

Comment:
again, try to avoid type coercion. if you merge in the latest develop, you can use TS 3.7 [nullish coalescing](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#nullish-coalescing): `this.props.selectedItem ?? this.state.selectedItem`